smithy-aws-traits/src/main/resources/META-INF/smithy/aws.api.smithy

```diff
@@ -201,6 +201,59 @@ structure service {
     endpointPrefix: String
 }
 
+/// Annotates a service as having tagging on 1 or more resources and associated
+/// APIs to perform CRUD operations on those tags
+@trait(selector: "service")
+structure tagEnabled {
```

Comment:
Can we mark tagging traits as unstable until we ship stuff with them or get closer to shipping stuff with them? I just want them to bake a bit.